src/azure-cli/azure/cli/command_modules/cosmosdb/custom.py

```diff
@@ -615,6 +676,19 @@ def cli_cosmosdb_mongodb_database_create(client,
                                                   mongodb_database_resource)
 
 
+def cli_cosmosdb_mongodb_database_exists(client,
+                                         resource_group_name,
+                                         account_name,
+                                         database_name):
+    """Checks if an Azure Cosmos DB MongoDB database exists"""
+    try:
+        client.get_mongo_db_database(resource_group_name, account_name, database_name)
+    except CloudError as ex:
+        return _handle_exists_exception(ex)
```

Comment:
By the way, you can add exception_handler kwargs in command for exception handler as shown in https://github.com/Azure/azure-cli/blob/master/src/azure-cli/azure/cli/command_modules/storage/commands.py#L231.

Reply:
I tried this, but I do not think I can return "True" in that case like I do in the exception handler here. If I do that, I get:

cli.azure.cli.core.util : 'CommandResultItem' object is not iterable

Is it possible to return something other than an exception from the exception handler kwags method?